src/setupTest.js

```diff
@@ -2,16 +2,23 @@
 
 import 'regenerator-runtime/runtime';
 import axios from 'axios';
-import Enzyme from 'enzyme';
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
 import { getAuthenticatedHttpClient, getAuthenticatedUser } from '@edx/frontend-platform/auth';
 
-Enzyme.configure({ adapter: new Adapter() });
-
 jest.mock('@edx/frontend-platform/auth');
 getAuthenticatedHttpClient.mockReturnValue(axios);
 getAuthenticatedUser.mockReturnValue({ administrator: false });
 
+// https://github.com/wwayne/react-tooltip/issues/595#issuecomment-638438372
+jest.mock('react-tooltip/node_modules/uuid', () => ({
+  v4: () => '00000000-0000-0000-0000-000000000000',
+}));
+// Related issue: https://github.com/jpuri/react-draft-wysiwyg/issues/780
```

Comment:
[curious] You link to `react-draft-wysiwyg` but mock `tinymce-react`. Why? Is there some interdependency?

Reply:
I can update the comment. There is no interdependency. The linked issue mentions the key function mocking to fix the snapshot issue in said package and this mock behavior was also needed for tinymce-react.